documentation/ntfs/ntfs_fare_extention.md

```diff
@@ -0,0 +1,69 @@
+This document describes an extension of the core [NTFS (fr)](./ntfs_fr.md) data format to add fares and tickets informations into Navitia.
+The [old fare extension](./ntfs_fare_extension_fr_deprecated.md) is still in use in conjunction with the files described here for backward compatibility matters.
+
+# Data format
+The data format of the files are the same as the core [NTFS ones](./ntfs_fr.md). In short, files are CSV encoded in utf8 and with a comma `,` separator.
+
+# Extension description
+## Added files of this extension
+
+File | Constraint | Comment
+--- | --- | ---
+tickets.txt | Required | Contains tickets declaration.
+ticket_prices.txt | Required | Contains prices of a ticket.
+ticket_uses.txt | Required | Contains where the ticket can be used
+ticket_use_perimeters.txt | Required | Defines the networks and lines of a ticket_use
+ticket_use_restrictions.txt | Optionnal | Defines a use of a ticket between zones or stop_areas
+
+## Other modifications
+The `object_properties.txt` and `object_codes.txt` files are extended to enable the reference of a ticket_id.
+
+# Added files descriptions
+
+## tickets.txt (Required)
+Field | Type | Constraint | Description
+--- | --- | --- | ---
+ticket_id | Text | Required | Unique ID of the ticket
+ticket_name | Text | Required | Name of the ticket (may be displayed to passengers)
+ticket_comment | Text | Optionnal | Comment on the ticket that may be displayed to the passenger
+
+## ticket_prices.txt (Required)
+Field | Type | Constraint | Description
+--- | --- | --- | ---
+ticket_id | Text | Required | Unique ID of the ticket
+ticket_price | Non-negative Float | Required |
+ticker_currency | Currency Code | Required | An ISO 4217 alphabetical currency code. For the list of current currency, refer to https://en.wikipedia.org/wiki/ISO_4217#Active_codes.
+ticket_validity_start | Date | Required | First day of usability of the ticket
+ticket_validity_end | Date | Required | Last day of usability of the ticket
+
+## ticket_uses.txt (Required)
+Field | Type | Constraint | Description
+--- | --- | --- | ---
+ticket_id | Text | Required | Unique ID of the ticket
+ticket_use_id | Text | Required | Unique ID of the ticket use
+max_transfers | integer | Required | Max number of transfers using one ticket. The value can be empty to indicate there is no constraint, but the field must be provided.
+boarding_time_limit | integer | Required | Specifies the time frame (in minutes) in which a passenger can board in a new vehicle. The value can be empty to indicate there is no constraint, but the field must be provided.
+alighting_time_limit | integer | Required | Specifies the time frame (in minutes) in which a passenger must get of the vehicle. When this duration is expired, the ticket is no longer valid. The value can be empty to indicate there is no constraint, but the field must be provided.
+
+## ticket_use_perimeter.txt (Required)
+
+A ticket_use must contain at least one included line or network.
```

Comment:
Minor: What can be a bit messy is things like "usable on all RATP network except line M1" : getting multiple lines on the same ticket can be a burden to read/edit (if randomly sorted).
Also, At some point we will probably need to add modes (but it's possible from current proposition).

Reply:
You're right, the provided fare data needs to be accurate, but that's also the case for stop_times ;-)
For the modes, I rejected this for when we will merge 2 datasets with a different price for a metro, both fares would apply on all the metro lines.